frontend/src/app/components/routes/main-view/components/menu/components/menu-item/menu-item.component.scss

```diff
@@ -22,8 +21,8 @@
         }
     }
     &.active {
```

Comment:
Swoją drogą nie powinno zostać przerobione na `--active`?

Reply:
No to nie jedyne miejsce, które powinno zostać przerobione
Staram się raczej poprawnie implementowac BEMa, Bartek pewnie też, ale takie rzeczy pewnie wychodzą przypadkiem no i trzeba by chyba po prostu znowu zrobić zadanie na poprawę wszystkich scss i wyłapaniu takich rzeczy

Reply:
Kilka poprawiłem w tym zadaniu, ale zostało jeszcze trochę

Reply:
Ddoałem zadanko na różne tego typu rzeczy CMS-138